lib/si-firecracker/src/firecracker.rs

```diff
@@ -61,6 +62,8 @@ impl FirecrackerJail {
     }
 
     pub async fn clean(id: u32) -> Result<()> {
```

Comment:
My gut says to cfg out the whole function... wait isn't the file cfg'd out?

Reply:
If I cfg out the func or the file, I I think need to slap cfgs all over the place in `local_uds.rs` where this gets called. This was the lightest touch I could figure out, but I'm open to cleaner approaches.